cli/internal/client/client.go

```diff
@@ -68,6 +68,16 @@ func NewClient(baseURL string, logger hclog.Logger, turboVersion string, teamID
 	return client
 }
 
+// IsLoggedIn returns true if this ApiClient has a credential (token)
+func (c *ApiClient) IsLoggedIn() bool {
+	return c.Token != ""
```

Comment:
I don't think we want to do any validation. It's ultimately a user-supplied string, via commandline or file. We should accurately report `403`s though.

Reply:
At some point after we have things working there are multiple days of just writing error messages to make our experience more-friendly.

(I personally like the idea of giving each error message an ID number so that it's easy to find when people copy and paste it into Google. Also, it's easy for us to track and sort error message IDs by frequency to identify pain points.)